### PR TITLE
Change summary, description and category

### PR DESCRIPTION
**Pull request by pmaier1**

### CODE REVIEW

**Comment by pmaier1** [comment]
@hurradieweltgehtunter

File: appinfo/info.xml [hunk 1]
```diff
@@ -2,10 +2,18 @@
 <info>
     <id>contacts</id>
     <name>Contacts</name>
-    <summary>A contacts app for ownCloud</summary>
+    <summary>Manage your CardDAV contacts from a feature-rich web-interface</summary>
     <description>
-The ownCloud contacts app is a user interface for ownCloud's CardDAV server.
-Easily sync contacts from various devices with your ownCloud and edit them online.</description>
+Contacts is a user interface for ownCloud's CardDAV server. ownCloud Server comes with a CardDAV component that allows users to synchronize and backup their address books with ownCloud and across multiple devices.
+Thanks to this feature users are provided with an alternative to proprietary contact synchronization services giving them back control over their data once more.
```
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: Proposal:
"Thanks to this feature users are provided with an alternative to proprietary contact synchronization ..."
-> "It provides  an alternative to proprietary contact synchronization ..."

File: appinfo/info.xml [hunk 2]
```diff
@@ -2,10 +2,18 @@
...
+
+The Contacts application is an extension to ownCloud's CardDAV interface. It equips ownCloud with a feature-rich frontend for the CardDAV server component, supports multiple address books and empowers users to easily manage their contacts from a web interface for ownCloud.
```
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: kind of double information to line 7/8

File: appinfo/info.xml [hunk 3]
```diff
@@ -2,10 +2,18 @@
...
+Users may find the URL for device synchronization (to configure synchronization in iOS, Android or other CardDAV-capable applications) in the 'Settings' section on the bottom left inside the application (link symbol).
+Furthermore users can share address books with other users or groups (read-only/editable) enabling to synchronize address books not only across devices but also across users.
+In addition to managing synchronized contacts the application also offers a feature to import contacts from vCards (.vcf).
+
+### Resources
+- [ownCloud Contacts & Calendar Documentation](https://doc.owncloud.org/server/10.0/user_manual/pim/index.html)</description>
```
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: please use tags for docs and remove from description
```
<documentation>
  <user>add link to user docs here</user>
  </admin>add_link to admin docs here</admin>
  <developer>add link to ev docs here</developer>
</documentation>
```
leave those empty where no link is available

**Review by hurradieweltgehtunter — CHANGES_REQUESTED** [review]

File: appinfo/info.xml (hunk 2 above)
Comment (pmaier1) [reply appinfo/info.xml]: no, it's a separation between the server component and the capabilities of the app

File: appinfo/info.xml (hunk 1 above)
Comment (pmaier1) [reply appinfo/info.xml]: why?

File: appinfo/info.xml (hunk 3 above)
Comment (pmaier1) [reply appinfo/info.xml]: We agreed to add docs to the description, wtf? Are those tags read and displayed by the marketplace?

Comment (hurradieweltgehtunter) [reply appinfo/info.xml]: yes. newly added feature

File: appinfo/info.xml (hunk 1 above)
Comment (hurradieweltgehtunter) [reply appinfo/info.xml]: just a proposal. "Thanks to" is used to show gratitude which in this case is maybe too much.

**Review by hurradieweltgehtunter — APPROVED** [review]

File: appinfo/info.xml [hunk 4]
```diff
@@ -2,19 +2,24 @@
...
+Furthermore users can share address books with other users or groups (read-only/editable) enabling to synchronize address books not only across devices but also across users.
+In addition to managing synchronized contacts the application also offers a feature to import contacts from vCards (.vcf).</description>
     <licence>AGPL</licence>
     <author>Hendrik Leppelsack, Thomas Müller, Alexander Weidinger, Jan-Christoph Borchardt, Tom Needham</author>
     <version>1.5.2</version>
     <documentation>
-    	<user>https://doc.owncloud.org/server/10.0/user_manual/pim/contacts.html</user>
-    	<admin>https://doc.owncloud.org/server/10.0/admin_manual/configuration_server/occ_command.html?highlight=contact#dav-commands</admin>
+    	<user>https://doc.owncloud.org/server/10.0/user_manual/pim/index.html</user>
+    	<admin>https://doc.owncloud.com/server/10.0/admin_manual/configuration/server/occ_command.html?highlight=contacts#dav-commands</admin>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: com vs org?

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: appinfo/info.xml (hunk 4 above)
Comment (pmaier1) [reply appinfo/info.xml]: changed

**Comment by DeepDiver1975** [comment]
Ill Squash and merge later.  Thx

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**